README.CentOS7.md

```diff
@@ -0,0 +1,152 @@
+
+```bash
+cp docs/config/apache-site /etc/httpd/conf.d/apache-site.conf
+```
+
+Customize and Verify your settings: 
+* Paths and settings in `/etc/httpd/conf.d/apache-site.conf` should be populated appropriately for your server. Replace placeholders such as `%LORISROOT%` with `/var/www/loris`, `%PROJECTNAME%` with `loris`, `%LOGDIRECTORY%` with `/var/log/httpd/loris-error.log` 
+ * DocumentRoot should point to `/var/www/loris/htdocs`
+ * The `smarty/templates_c/` directory must be writable by Apache (e.g. by running: `chmod 775 /var/www/loris/smarty/templates_c)
+
+Finally, restart apache:
+```
+sudo service httpd restart
+sudo service httpd status
+```
+# Install LORIS
+
+For the purpose of following LORIS conventions and easy understanding of all LORIS documentation, we recommend the following account names: 
+
+* `lorisadmin` unix user with sudo permission who will setup and manage the LORIS back-end
+* `lorisuser` MySQL database user with limited (insert, delete, update, select...) permissions, for database transactions requested by the Loris front-end
+* `admin` default front-end username for LORIS administrator account (login via browser)
+
+## Run the LORIS install script to set up directories
+
+```
+cd /var/www/loris/tools
+./install.sh
+```
+Run the makefile (use `make dev` if you are setting up a development sandbox)
+```
+cd /var/www/$projectname
+make
+```
+## Install your database
+Open your browser and go to: `http://%IPADDRESS%/installdb.php`
+
+(%IPADDRESS% will likely be the IP address of the VM you are ssh'ed into)
+
+The web page will prompt you for the following information: 
+
+ * `Server Hostname` localhost if your database if hosted on your VM or the IP address of your database server
+
+ * `Admin Username` should be `root` or a unix user with a high level of privilege on your VM
+
+ * `Admin Password` whatever password you set when running the `mysql_secure_installation` step
+
+ * `Database Name` set to "LORIS" by default, can be customised
+
+Click submit, and on the next screen that is presented, follow instructions to enter the username and password of your backend user (`lorisuser`), and front-end `admin` user. 
+
+If you encounter issues creating/generating your config file, you may have to manually paste the xml output that appears on the screen into the file `/var/www/loris/project/config.xml`
+
+Your LORIS instance should now be accessible by pointing your browser URL to `http://%IPADDRESS%`
+
+If there are any errors or you get a blank page, troubleshoot the errors in your apache error log - by default 
+ `/var/log/httpd/loris-error.log`
+
+**Next:** Follow the [**Setup Guide** in the LORIS wiki](https://github.com/aces/Loris/wiki/Setup) for all post-install steps and troubleshooting.  
+Config settings can be accessed via the front-end Config module under the Admin drop-down menu.
```

Comment:
SHould not it have the same `Community`, `Contributing` and `Powered by MCIN` section as main README.md?

Reply:
I don't think it needs these fields. The only reason `README.md` has this is because it's doing double duty as a landing page for the repo.